src/ui/ResponsiveComponents/ResponsiveRecommendation.ts

```diff
@@ -173,51 +168,39 @@ export class ResponsiveRecommendation implements IResponsiveComponent {
   }
 
   private getFacetSliders(): any[] {
-    let facetSliders = [];
-    _.each(this.coveoRoot.findAll('.' + Component.computeCssClassNameForType(`FacetSlider`)), facetSliderElement => {
-      let facetSlider = Component.get(facetSliderElement);
-      facetSliders.push(facetSlider);
+    const facetSliders = [];
+    this.coveoRoot.findAll(Component.computeSelectorForType('FacetSlider')).forEach(facetSliderElement => {
+      const facetSlider = Component.get(facetSliderElement);
+      facetSlider && facetSliders.push(facetSlider);
```

Comment:
I'm not clear on the key change that solves the bug. Was it this check for truthiness?

If so, I'm curious why was there no FacetSlider class instance returned by `Component.get`?

Reply:
FacetSlider was not an issue AFAIK but I still added the check.
The facets were returning undefined when the mode was responsive. Probably due to the fact that they are move into the dropdown in that case, so getting them each time ensures that we get the most updated facets rather than the ones that were there at the initialization of the component. 
Again, this issue only popped up using minified Lazy code